modules/system/traits/ConfigMaker.php

```diff
@@ -67,8 +67,19 @@ public function makeConfig($configFile = [], $requiredConfig = [])
 
             $config = Yaml::parseFile($configFile);
 
-            /*
-             * Extensibility
+            /**
+             * @event system.extendConfigFile
+             * Provides an opportunity to modify config files
+             *
+             * Example usage:
+             *
+             *     Event::listen('system.extendConfigFile', function ((string) $path, (array) $config) {
```

Comment:
@mjauvin Should `$path` be `$publicFile`, as that's what you use in the line below?
```suggestion
             *     Event::listen('system.extendConfigFile', function ((string) $publicFile, (array) $config) {
```

Reply:
@bennothommo I suggested $path because it's more clear to the end user what that value is

Reply:
@LukeTowers ah fair enough. @mjauvin You'll just need to change the line below to reflect then :)